Assets/Scripts/UI/Canvas/PhotoTaking/CameraPanel.cs

```diff
@@ -13,6 +13,7 @@ public sealed class CameraPanel : CameraPermission
         [SerializeField] private RawImage background;
         [SerializeField] private TMP_Text pointOfInterestName;
         [SerializeField] private Texture2D pointOfInterestRender;
+        [SerializeField] private RenderTexture renderTexture;
```

Comment:
Deze nieuwe variabel is toegevoegd, maar wat gebeurt er nu met de background variabel? We hebben ze niet allebei nodig toch?

Reply:
Background is de reference naar de rawimage op de panel die geactiveerd word, renderTexture is zodat de camera naar een texture gerendered wordt zodat het gebruikt kan worden in de shader